src/web3/SmartContractsClient.ts

```diff
@@ -184,7 +184,7 @@ export class SmartContractsClient
   ): Promise<IContractReadOperationResponse> {
     // request data
     const data = {
-      max_gas: readData.maxGas,
+      max_gas: Number(readData.maxGas),
```

Comment:
the readme is saying that max gas is expressed in BigInt, why here you concert to Number?

Reply:
because that data is the payload I am sending to the node over json-rpc. it needs to be a number (u64 in rust), otherwise the call fails.